tests/unit/swagger/schemaDetails3.spec.ts

```diff
@@ -1343,4 +1343,24 @@ describe('Definition generation for OpenAPI 3.0.0', () => {
       });
     });
   });
+
+  describe('module declarations with namespaces', () => {
+    it('should generate the proper schema for a model declared in a namespace in a module', () => {
+      const ref = specDefault.spec.paths['/GetTest/ModuleRedeclarationAndNamespace'].get?.responses['200'].content?.['application/json']['schema']['$ref'];
+      expect(ref).to.equal('#/components/schemas/TsoaTest.TestModel73');
+      expect(specDefault.spec.components.schemas?.['TsoaTest.TestModel73']).to.deep.include({
```

Comment:
There's a helper you can use here: `getComponentSchema('TsoaTest.TestModel73')`
Also, is there a reason you're not using equals here?

Reply:
Fixed. There wasn't really a reason. I just wanted to skip properties that were undefined, but it doesn't really make a difference.